wasm/Makefile

```diff
@@ -1,6 +1,11 @@
 CPPFLAGS = -I. -Iems_helper -I../common -Wall -fno-exceptions -fno-rtti -s BINARYEN=1
 LDFLAGS = --embed-file ../assets@/
 
+OUT_DIR = build
```

Comment:
I'd prefer we output to a build folder in the root of the repo, with subfolders for each target: //build/wasm, //build/android and //build/webgl